src/components/pager/js/Pager.js

```diff
@@ -295,8 +295,12 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     });
 
     fluid.pager.previousNext.update = function (that, disabledStyle, newModel) {
-        that.previous.toggleClass(disabledStyle, newModel.pageIndex === 0);
-        that.next.toggleClass(disabledStyle, newModel.pageIndex === newModel.pageCount - 1);
+        var previousToggle = (newModel.pageIndex === 0);
+        var nextToggle = (newModel.pageIndex === newModel.pageCount - 1);
+        that.previous.toggleClass(disabledStyle, previousToggle);
+        that.next.toggleClass(disabledStyle, nextToggle);
+        that.previous.children('a').attr('aria-disabled', previousToggle);
+        that.next.children('a').attr('aria-disabled', nextToggle);
```

Comment:
Also we use `"` not `'` around strings. Please make sure to update this and run `grunt lint` to verify that there are no other code style issues.